src/validators/email.js

```diff
@@ -3,4 +3,4 @@ import i18n from '@dhis2/d2-i18n'
 export const emailPattern = /^.+@.+\.[a-zA-Z]+$/
 export const invalidEmailMessage = i18n.t('Not a valid e-mail')
 export const email = value =>
-    emailPattern.test(value) ? undefined : invalidEmailMessage
+    value && !emailPattern.test(value) ? invalidEmailMessage : undefined
```

Comment:
I don't think this is right actually. If you have a required email field, you should have this validate prop:
```javascript
validate={composeValidators(required, email)}
```
This change above will make it impossible to validate an optional email field.